src/dvr/dvr_config.c

```diff
@@ -937,77 +937,77 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_BOOL,
       .id       = "day-dir",
-      .name     = N_("Make Subdirectories Per Day"),
+      .name     = N_("Make subdirectories per day"),
       .off      = offsetof(dvr_config_t, dvr_dir_per_day),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "channel-dir",
-      .name     = N_("Make Subdirectories Per Channel"),
+      .name     = N_("Make subdirectories per channel"),
       .off      = offsetof(dvr_config_t, dvr_channel_dir),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "title-dir",
-      .name     = N_("Make Subdirectories Per Title"),
+      .name     = N_("Make subdirectories per title"),
       .off      = offsetof(dvr_config_t, dvr_title_dir),
       .group    = 4,
     },
     {
       .type     = PT_BOOL,
       .id       = "channel-in-title",
-      .name     = N_("Include Channel Name In Filename"),
+      .name     = N_("Include channelname In filename"),
```

Comment:
Just to be obtuse (sorry!), I wouldn't use "channelname" as it's not a widely-used compound noun like "filename" IMO.